Source/WebCore/page/EventHandler.cpp

```diff
@@ -288,8 +288,7 @@ class SyntheticTouchPoint : public PlatformTouchPoint {
         m_id = idDefaultValue; // There is only one active TouchPoint.
         m_screenPos = event.globalPosition();
         m_pos = event.position();
-        m_radiusY = radiusYDefaultValue;
-        m_radiusX = radiusXDefaultValue;
+        m_radius = WebCore::FloatSize(radiusYDefaultValue, radiusXDefaultValue);
```

Comment:
```suggestion
        m_radius = WebCore::FloatSize(radiusXDefaultValue, radiusYDefaultValue);
```

It looks like these are flipped?

Reply:
Good catch, fixed in latest commits